test/jdk/ProblemList.txt

```diff
@@ -743,6 +743,8 @@ sun/tools/jstatd/TestJstatdRmiPort.java                         8251259,8293577
 jdk/incubator/vector/ShortMaxVectorTests.java                   8306592 generic-i586
 jdk/incubator/vector/LoadJsvmlTest.java                         8305390 windows-x64
 
+sun/misc/SunMiscSignalTest.java                                 8370207 generic-all
```

Comment:
The sun/misc tests are in jdk_lang rather than jdk_other but okay to put it here temporarily.

Reply:
Thanks Alan. I couldn't see anything to guide the placement. It will be gone again in 24 hours. :)